### ISSUE TITLE
Broken build latest v5.2.0 for Windows?

### ISSUE THREAD

**Issue by jgliss** [conda-forge/proj.4-feedstock#47]
Issue: Referring to conda-forge/geonum-feedstock#6: I think the latest build of v5.2.0 for Windows (h6538335_1002) has a problem with  respect to where shared data files are getting installed. I installed into a new env:

```
conda create -n proj4_520_cf_test python=3 proj4=5.2.0
conda activate proj4_520_cf_test
```

Shared datafiles (e.g. `epsg`, required by `basemap`) are installed directly into dir:

*Anaconda3\envs\proj4_520_cf_test\Library\share*

Shouldn't they rather be installed into:

*Anaconda3\envs\proj4_520_cf_test\Library\share\proj*

(which is also where basemap is looking for it)?

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# Name                    Version                   Build  Channel
ca-certificates           2019.3.9             hecc5488_0    conda-forge
certifi                   2019.3.9                 py37_0    conda-forge
openssl                   1.1.1b               hfa6e2cd_2    conda-forge
pip                       19.1.1                   py37_0    conda-forge
proj4                     5.2.0             h6538335_1002    conda-forge
python                    3.7.3                hb12ca83_0    conda-forge
setuptools                41.0.1                   py37_0    conda-forge
sqlite                    3.28.0               hfa6e2cd_0    conda-forge
vc                        14.1                 h21ff451_3    anaconda
vs2015_runtime            15.5.2                        3    anaconda
wheel                     0.33.4                   py37_0    conda-forge
wincertstore              0.2                   py37_1002    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : proj4_520_cf_test
    active env location : C:\Users\Jonas\Anaconda3\envs\proj4_520_cf_test
            shell level : 3
       user config file : C:\Users\Jonas\.condarc
 populated config files : C:\Users\Jonas\.condarc
          conda version : 4.6.14
    conda-build version : 3.18.1
         python version : 3.7.3.final.0
       base environment : C:\Users\Jonas\Anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/anaconda/win-64
                          https://conda.anaconda.org/anaconda/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\Jonas\Anaconda3\pkgs
                          C:\Users\Jonas\.conda\pkgs
                          C:\Users\Jonas\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\Jonas\Anaconda3\envs
                          C:\Users\Jonas\.conda\envs
                          C:\Users\Jonas\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.6.14 requests/2.22.0 CPython/3.7.3 Windows/10 Windows/10.0.17134
          administrator : False
             netrc file : None
           offline mode : False
```
</details>

**Comment by xylar** [comment]
Probably from https://github.com/conda-forge/proj.4-feedstock/commit/3d75fdd2dd4a980373501055e7e743746a7205bb

@ocefpaf, do you have a sense of what went wrong? I knew windows wasn't building in my PR before that but it didn't need a new build.

**Comment by snowman2** [comment]
The files were moved to share/proj in 6.0.0. Before that, if was just share.

**Comment by snowman2** [comment]
See: https://github.com/OSGeo/proj.4/pull/1274

**Comment by ocefpaf** [comment]
> Probably from [3d75fdd](https://github.com/conda-forge/proj.4-feedstock/commit/3d75fdd2dd4a980373501055e7e743746a7205bb)
> 
> @ocefpaf, do you have a sense of what went wrong? I knew windows wasn't building in my PR before that but it didn't need a new build.

Don't worry about that PR. The Windows builds were broken before that. The original PR that added `5.2.0` passed on the PR but failed on master.

> The files were moved to share/proj in 6.0.0. Before that, if was just share.

Yep. It was just share and it worked. I'm not exact sure what the issue is. Still investigating.

Note that it is hard to keep old package alive. We could use a PR that updates `basemap` to the latest `proj.4 6` or a PR that changes `geonum` to use `cartopy` (which will be `proj.4 6` ready sooner than `basemap`.)

**Comment by jgliss** [comment]
Changing from `basemap` to `cartopy` has long been on my list for `geonum`. However, due to limited time (I only work on it in my spare time), I cannot promise when I will be able to do that... would be very happy for a temporary fix, that allows to install basemap on Windows using `conda-forge`.

**Comment by ocefpaf** [comment]
I'm investigating the possibility of re-build `basemap` with an older `proj.4` (`5.1.0`?) but that will take a while.

**Comment by xylar** [comment]
If we label the recent windows builds as broken, wouldn't conda-forge revert to the previous, working build?

**Comment by ocefpaf** [comment]
> If we label the recent windows builds as broken, wouldn't conda-forge revert to the previous, working build?

I already did that but `basemap` is pinned to `proj.4 5.2.0` and there are no working `conda-forge` builds of those.

**Comment by ocefpaf** [comment]
The problems seems to be that I forgot to revert a change in the scripts when adding the `v5.2.0` branch. Should be fixed now in #48
